crates/astria-composer/src/searcher/mod.rs

```diff
@@ -297,14 +297,14 @@ impl Searcher {
                         Err(e) => Err(e),
                     }
                 }
-                .map(|fut| (chain_id.clone(), fut))
+                .map(|fut| (rollup_name.clone(), fut))
             })
             .collect::<FuturesUnordered<_>>();
-        while let Some((chain_id, maybe_err)) = statuses.next().await {
+        while let Some((rollup_name, maybe_err)) = statuses.next().await {
             if let Err(e) = maybe_err {
                 return Err(e).wrap_err_with(|| {
                     format!(
-                        "collector for chain ID {chain_id} failed while waiting for it to become \
+                        "collector for rollup {rollup_name} failed while waiting for it to become \
```

Comment:
```suggestion
                        "collector for rollup `{rollup_name}` failed while waiting for it to become \
```
I like doing this to have the name stand out a bit more.